### PR TITLE
Enhanced install instructions

### PR DESCRIPTION
**Pull request by cuducos**
Regarding #19, I decided to answer what was asked with this new _pull request_.

You've guys improved a lot starting this new version of the `README.md` — thanks for that! It is way better than before ; )

Personally I suggests this _pull request_ because:

* I like very detailed instructions, full of links (if installing _whatever_ fails, you have the _wahetever_'s official website already linked to look for details about installing  or configuring _whatever_ in your own platform and setting)
* The same is valid for Django setting variables (I linked all Django 1.9 docs; the topics linked haven't changed I guess, and we might update Pets in a near future, right?)
* From my experience, I added some bits of installing stuff in a Mac
* And I edited a bit of the markdown to try to make the document more organized and user friendly

I do understand anyone who criticize my version on `README.md` as too verbose — so feel free to dump this _pull request_ hahaha…

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,110 @@
...
 cp contrib/sample-env pets/.env
+```
+
+Then you have to set following variables.
+
+#### Basic Django settings
+
+* **`SECRET_KEY`**: [Django's secret key](https://docs.djangoproject.com/en/1.9/ref/settings/#std:setting-SECRET_KEY)
+* **`ALLOWED_HOSTS`** (e.g. `127.0.0.1, .localhost`) [Django's allowed hosts](https://docs.djangoproject.com/en/1.9/ref/settings/#allowed-hosts)
+* **`DJANGO_SETTINGS_MODULE`**: In order to make development and deploy to production simpler there's two settings module; `pets.settings.dev` for development and `pets.settings.dev` for production.
```
Comment (dirtycoder) [inline README.md]: `pets.settings.prod` for production. :)

File: README.md [hunk 2]
```diff
@@ -1,44 +1,110 @@
...
+The project configuration uses [python-decouple](https://pypi.python.org/pypi/python-decouple/) to dynamically read environment variables and `.env` files.
 
-1. Fork and clone the repository;
-2. Configure your instance (python-decouple);
-   * Remember to configure the username, password and database in your postgreSQL. 
-3. Install Pillow dependencies;
-4. Install PhantonJS, see the julionc [tutorial](https://gist.github.com/julionc/7476620);
-5. Install project requirements;
-6. Execute all tests, it will take some minutes. 
+If you want, you can get startted by copying `contrib/sample-env` as `.env`:
```
Comment (dirtycoder) [inline README.md]: "started"

File: README.md [hunk 3]
```diff
@@ -1,44 +1,110 @@
...
 
 ```console
-git clone https://github.com/<username>/pets.git && cd pets
 cp contrib/sample-env pets/.env
+```
+
+Then you have to set following variables.
```
Comment (dirtycoder) [inline README.md]: "set the following" :question:

File: README.md [hunk 4]
```diff
@@ -1,44 +1,110 @@
...
+
+#### Basic Django settings
+
+* **`SECRET_KEY`**: [Django's secret key](https://docs.djangoproject.com/en/1.9/ref/settings/#std:setting-SECRET_KEY)
```
Comment (dirtycoder) [inline README.md]: As the project still uses the LTS version (1.8) what do you think about using the corresponding version of the docs?

File: README.md [hunk 5]
```diff
@@ -1,44 +1,110 @@
...
+* **`ALLOWED_HOSTS`** (e.g. `127.0.0.1, .localhost`) [Django's allowed hosts](https://docs.djangoproject.com/en/1.9/ref/settings/#allowed-hosts)
```
Comment (dirtycoder) [inline README.md]: The same here about the docs version.

File: README.md [hunk 6]
```diff
@@ -1,44 +1,110 @@
...
+* **`DJANGO_SETTINGS_MODULE`**: In order to make development and deploy to production simpler there's two settings module; `pets.settings.dev` for development and `pets.settings.dev` for production.
+
+#### Database
+
+* **`DB_NAME`**, **`DB_USERNAME`**, **`DB_PASSWORD`**, **`DB_IP`**, **`DB_PORT`**: Credentials to access Pets's database.
+* **`DB_ENGINE`**: Django databse backend to connect to Pets's database (e.g. `django.db.backends.postgresql_psycopg2`)
+* **`DB_CONN_MAX_AGE`**: (e.g. `0`) [Django's database `CONN_MAX_AGE`](https://docs.djangoproject.com/en/1.9/ref/settings/#std:setting-CONN_MAX_AGE`)
```
Comment (dirtycoder) [inline README.md]: Docs version here too. :)

File: README.md [hunk 7]
```diff
@@ -1,44 +1,110 @@
...
+#### OAuth
+
+If you want to login via social media, you will have to create apps as a developer [at Facebook](https://developers.facebook.com) and/or [Twitter](https://apps.twitter.com). Once you're done, set the _app secret_ and _app key_ for each of them:
+
+* `SOCIAL_AUTH_FACEBOOK_KEY`
+* `SOCIAL_AUTH_FACEBOOK_SECRET`
+* `SOCIAL_AUTH_TWITTER_KEY`
+* `SOCIAL_AUTH_TWITTER_SECRET`
+
+While setting you app there you should add the proper redirect URLs.
```
Comment (dirtycoder) [inline README.md]: "your"

**Comment by dirtycoder** [comment]
@cuducos I've made a few suggestions, thanks for the PR, this looks awesome!

File: README.md (hunk 7 above)
Comment (cuducos) [inline README.md]: BTW which are the return URLs?

**Comment by cuducos** [comment]
Thanks for the review ; ) Fixed.

File: README.md (hunk 7 above)
Comment (dirtycoder) [inline README.md]: To be honest, I'm not sure what are you talking about. :laughing: 
If it's about the "callback", I'm using the default provided by `python-social-auth`, for reference, [here](http://psa.matiasaguirre.net/docs/configuration/settings.html#urls-options)

Maybe we could remove remove this line altogether, what do you think?

**Comment by dirtycoder** [comment]
One more thing, as the commits are close related, may I ask you to rebase both in a single commit?

**Comment by cuducos** [comment]
@dirtycoder If you haven't set one, it's not needed them. Gonna remove and rebase.

**Comment by cuducos** [comment]
One commit to rule them all 315c894

**Comment by cuducos** [comment]
I'm starting the API (#18) and I've seen that this wasn't merged into `next`. Would you do it? (Just asking to avoid merging it locally and causing conflicts in the next PRs).

**Pull request merged**